website/docs/r/iothub.html.markdown

```diff
@@ -0,0 +1,92 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_iothub"
+sidebar_current: "docs-azurerm-resource-iothub"
+description: |-
+  Creates a new IotHub resource 
+---
+
+# azurerm_iothub
+
+Creates a new IotHub
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "resourceGroup1"
+  location = "West US"
+}
+
+
+resource "azurerm_iothub" "test" {
+	name                             = "test"
+	resource_group_name              = "${azurerm_resource_group.test.name}"
+	location                         = "${azurerm_resource_group.test.location}"
+	sku {
+		name = "S1"
+		tier = "Standard"
+		capacity = "1"    
+	}
+
+	tags {
+		"purpose" = "testing"
+	    }
+	}
+
```

Comment:
_minor_ could we make the spacing consistent here by switching to use spaces instead of tabs?